src/trove/string-dict.scrbl

```diff
@@ -281,18 +285,12 @@ end
 ]
 
 Applies the function @pyret{f} to each key in the immutable string-dict, and
-returns a list of the returned values in some order.
+returns a list of the returned values in some unspecified order.
 
 @examples{
-fun one-of(ans, elts):
-  is-some(for find(elt from elts):
-    ans == elt
-  end)
-end
-
 check:
   sd2 = [string-dict: "a", 10, "b", 6]
-  sd2.map-keys(lam(x): string-append(x, x) end) is%(one-of)
+  sd2.map-keys(lam(x): string-append(x, x) end) is%(lam(lft, rt): rt.member(lft) end)
```

Comment:
This is complicated-looking.  Why not use `.sort()` like you did on line 273 above for `keys-list`?